include/core/truthTable/pla_parser.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "core/truthTable/truth_table.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] inline syrec::TruthTable::Cube::Value transform_pla_to_constants(const char& c) {
```

Comment:
this function does not seem to be appropriately named since it does not only return constants but also `true` and `false`.